### DISCUSSION TITLE
Possible to run on armv6?

### DISCUSSION THREAD

**Discussion by redthor** [nanomq/nanomq#1084] [Q&A]
Would it be possible to run it on armv6?
I have a raspberry pi zero w.

Cheers

**Comment by JaylinYu** [comment]
Sure for core broker functionalities, just do a cross-compile. 
But I am not sure for other parts which requires third-party dependencies

**Comment by redthor** [comment]
Thanks. I have got it compiled on armv6 with  https://github.com/dockcross/dockcross and

```
linux-armv6-musl -a "-e LDFLAGS='-static'" bash -c "cd build && cmake -DNNG_ENABLE_SQLITE=ON .. && make"
```

No TLS right now, might try that later.
